syntax = "proto3";

message Response {
    int32 code = 1; //状态码
    message data {
        string id = 1;
        string title = 2;
    }
    repeated data dataList =2 ;
}

message Request {
    string cate_id = 1;
    int32 page = 2;
    int32 pageSize = 3;
}

